waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -803,9 +809,10 @@
         ;; delay iff the log-bucket-url setting was given the scheduler config.
         log-bucket-sync-secs (if log-bucket-url (:log-bucket-sync-secs context) 0)
         total-sigkill-delay-secs (+ pod-sigkill-delay-secs log-bucket-sync-secs)
+        has-reverse-proxy? (get env "GRPC_TRANSCODER")
```

Comment:
Let's get this environment variables from a configuration (available in the context).

Reply:
Also, let's use `(contains? env "GRPC_TRANSCODER")`